tshirts.cpp

```diff
@@ -13,10 +13,17 @@ char size(int cms) {
     return sizeName;
 }
 
-int main() {
+void checkTshirtSizes()
+{
     assert(size(37) == 'S');

```

Comment:
the test code and production code are in same file. Please refactor to segregate them in separate files